src/lava/lib/optimization/solvers/generic/sub_process_models.py

```diff
@@ -0,0 +1,209 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+try:
+    from lava.magma.core.model.nc.net import NetL2
+except ImportError:
+    class NetL2:
+        pass
+
+import numpy as np
+from lava.lib.optimization.solvers.generic.hierarchical_processes import (
+    CostConvergenceChecker,
+    DiscreteVariablesProcess,
+    StochasticIntegrateAndFire)
+from lava.magma.core.decorator import implements, requires
+from lava.magma.core.model.sub.model import AbstractSubProcessModel
+from magma.core.model.py.model import PyLoihiProcessModel

```

Comment:
Should be lava.magma...

Interesting, why doesn't this throw an error in any of the unit tests?